src/integration.test.ts

```diff
@@ -105,14 +105,26 @@ describe("Registry end to end integration tests", () => {
 });
 
 describe("Upload and download integration tests", () => {
-  it("Should get file contents", async () => {
+  it("Should get plaintext file contents", async () => {
+    const fileData = "testing";
+
+    // Upload the data to acquire its skylink
+    const file = new File([fileData], dataKey, { type: "text/plain" });
+    const { skylink } = await client.uploadFile(file);
+
+    const { data } = await client.getFileContent(skylink);
+    expect(data).toEqual(expect.any(String));
+    expect(data).toEqual(data);
+  });
+
+  it("Should get JSON file contents", async () => {
     const json = { key: "testdownload" };
 
     // Upload the data to acquire its skylink
     const file = new File([JSON.stringify(json)], dataKey, { type: "application/json" });
```

Comment:
What happens if we don't specify the content type? Would it be detected?
No need to cover this now but it might be interesting to verify the behaviour.

Reply:
Follow-up: https://github.com/NebulousLabs/skynet-js/issues/241